### ISSUE TITLE
Rule Proposal: `vue/require-render-return`

### ISSUE THREAD

**Issue by armano2** [vuejs/eslint-plugin-vue#107]
## Please describe what the rule should do:

Enforces render function to always return value (require-render-return)

## What category of rule is this? (place an "X" next to just one item)
`[ ]` Enforces code style
`[x]` Warns about a potential error
`[ ]` Suggests an alternate way of doing something
`[ ]` Other (please specify:)

Provide 2-3 code examples that this rule will warn about:
```js
export default {
  render () {
  }
}
```
```js
export default {
  render (h) {
    if (foo) {
      return
    }
  }
}
```
